README.md

```diff
@@ -22,7 +22,8 @@ Ballerina also provides specially designed various database-specific DB connecto
 
 ### Client
 
-The database client should be created using any of the above-listed database libraries and once it is created, the operations and functionality explained below can be used.
+The database client should be created using any of the above-listed database libraries and kept open throughout 
+the entirety of the application to perform the operations and functionality which are explained below.
```

Comment:
```suggestion
A database client can be created using any of the above-listed database libraries. The operations and functionality explained below can be used with the newly created client.

> **Tip** : The client should be used throughout the application lifetime.
```

Reply:
Lets add another note to the Close the client section.

Reply:
`Handle connection pools` should also have a tip to configure this as per the need rather than using default values.